src/middlewares/isUserVerified.js

```diff
@@ -0,0 +1,12 @@
+import { users } from '../database/models';
+
+const isUserVerified = async (req, res, next) => {
+  const { email } = req.user;
+  const isUserExist = await users.findOne({ where: { email, isVerified: true } });
+
+  if (!isUserExist) {
+    return res.status(403).json({ error: 'Your email is not verified, please verify your email first' });
```

Comment:
Shouldn't it be `res.status(401)`? 
you are requesting user to verify email when you use `403` status code you are telling user it is forbidden to do it.